feature/tabsflow/favourites/src/main/java/com/epmedu/animeal/favourites/presentation/ui/FavouritesScreenUI.kt

```diff
@@ -197,8 +199,18 @@ private fun ScreenScaffold(
             onFeedingEvent(FeedingEvent.Start(it.id))
         })
     }
-    if (state.feedState.feedingConfirmationState == FeedingConfirmationState.FeedingStarted) {
-        navigator.navigateTo(TabsRoute.Home.name)
+    when (state.feedState.feedingConfirmationState) {
```

Comment:
You can separate this into separate "OnFeedingConfirmationState" function, as you did in SearchScreenUi.